components/Tabs2/src/private/MDCTabBarViewItemView.m

```diff
@@ -23,16 +23,110 @@
 /** The minimum height of any item view. */
 static const CGFloat kMinHeight = 48;
 
+/// Outer edge padding from spec: https://material.io/go/design-tabs#spec.
+static const UIEdgeInsets kEdgeInsets = {.top = 12, .right = 16, .bottom = 12, .left = 16};
+
+@interface MDCTabBarViewItemView ()
+
+@property(nonatomic, strong) UIView *contentView;
+@property(nonatomic, strong) UIImageView *iconImageView;
+@property(nonatomic, strong) UILabel *titleLabel;
+
+@end
+
 @implementation MDCTabBarViewItemView
 
+#pragma mark - Init
+
+- (instancetype)initWithFrame:(CGRect)frame {
+  self = [super initWithFrame:frame];
+  if (self) {
+    _title = @"";
+
+    self.isAccessibilityElement = YES;
+
+    // Create initial subviews
+    [self initSubviews];
+  }
+  return self;
+}
+
+#pragma mark - UIView
+
+- (void)layoutSubviews {
+  [super layoutSubviews];
+
+  CGRect contentFrame = UIEdgeInsetsInsetRect(self.bounds, kEdgeInsets);
+  self.contentView.frame = contentFrame;
+
+  CGSize iconSize = [self.iconImageView sizeThatFits:contentFrame.size];
+  CGSize labelSize = [self.titleLabel sizeThatFits:contentFrame.size];
+  CGFloat centerX = CGRectGetMidX(self.contentView.bounds);
+  CGFloat labelCenterY = CGRectGetMaxY(self.contentView.bounds) - labelSize.height / 2;
+  self.titleLabel.frame =
+      CGRectMake(centerX - labelSize.width / 2, labelCenterY - labelSize.height / 2,
+                 labelSize.width, labelSize.height);
+  CGFloat iconCenterY = CGRectGetMinY(self.titleLabel.frame) / 2;
+  self.iconImageView.frame =
+      CGRectMake(centerX - iconSize.width / 2, iconCenterY - iconSize.height / 2, iconSize.width,
+                 iconSize.height);
+}
+
+#pragma mark - UIAccessibility
+
+#pragma mark - Private
+
+- (void)initSubviews {
```

Comment:
Please rename this to begin with something other than "init" since that may be a little confusing.   Our team's convention has been to name this `common{$CLASS}Init`. In this case, `commonMDCTabBarViewItemViewInit`.

Please also be sure that the class is correctly initialized from `initWithCoder:`

Reply:
Done.